docs/specs/schemas/supporting-tools.md

```diff
@@ -4,9 +4,9 @@ This page outlines tools and services that currently support the development con
 
 While most [dev container properties](devcontainerjson-reference.md) apply to any supporting tool or service, a few are specific to certain tools, which are outlined below.
 
-## devcontainer CLI
+## Dev container CLI
 
-Given the growing number of use cases for dev containers, there is a companion [devcontainer command line interface](https://code.visualstudio.com/docs/remote/devcontainer-cli) (CLI) that can be used independent of the Remote - Containers extension or GitHub Codespaces.
+Given the growing number of use cases for dev containers, there is a companion dev container command line interface (CLI) that can be used independent of specific tools or editors. The publishing of this CLI is being discussed in an [issue](https://github.com/microsoft/dev-container-spec/issues/9).
```

Comment:
The way I think about it is that the CLI *is* the reference implementation. Not sure the wording here captures that, but fine otherwise.

Reply:
Yeah that makes sense. Updated to:

> There will be a dev container command line interface (CLI) that can take a `devcontainer.json` and create and configure a dev container from it.
> 
> The publishing of this CLI is being discussed in an [issue](https://github.com/microsoft/dev-container-spec/issues/9).